backend/api/sources.js

```diff
@@ -14,6 +14,14 @@ const apiSources = {
     trendingLink: "https://pixabay.com/api/?key=",
     limitString: "per_page",
     offsetString: "page"
+  },
+  unsplash: {
+    id: "unsplash",
+    key: process.env.UNSPLASH_API_KEY, // Your Unsplash API Key here
```

Comment:
Can you remove this comment here and other two api objects?  This comment is useless since we have .env files

Reply:
Do I have to remove only the API key comments from all three API objects?